### PR TITLE
New controller api

### PR DESCRIPTION
**Pull request by EthanVanderHorn**

### CODE REVIEW

File: nanome/_internal/_structure/_serialization/_workspace_serializer.py [hunk 1]
```diff
@@ -20,15 +20,15 @@ def name(self):
     def serialize(self, version, value, context):
         context.write_using_serializer(self.array, value._complexes)
 
-        context.write_using_serializer(self.vector, value._position)
-        context.write_using_serializer(self.quaternion, value._rotation)
-        context.write_using_serializer(self.vector, value._scale)
+        context.write_using_serializer(self.pos, value._position)
+        context.write_using_serializer(self.rot, value._rotation)
+        context.write_using_serializer(self.pos, value._scale)
```
Comment (ajm13) [inline nanome/_internal/_structure/_serialization/_workspace_serializer.py]: Is this intentionally using a `UnityPositionSerializer`?

File: nanome/api/plugin_instance.py [hunk 2]
```diff
@@ -198,13 +198,33 @@ def update_content(self, content):
 
     def update_node(self, node):
         """
-        | Update a layout node and its children
+        | Update the position, scale, and rotation of the menu
```
Comment (ajm13) [inline nanome/api/plugin_instance.py]: hmm...

File: nanome/api/plugin_instance.py [hunk 3]
```diff
@@ -323,6 +343,13 @@ def request_presenter_info(self, callback):
         id = self._network._send(_Messages.presenter_info_request)
         self._save_callback(id, callback)
 
+    def request_controller_transforms(self, callback):
+        """
+        | Requests presenter account info (head position, head rotation, left controller position, left controller rotation, right controller position, right controller rotation)
```
Comment (ajm13) [inline nanome/api/plugin_instance.py]: "account info"?

File: nanome/_internal/_structure/_serialization/_workspace_serializer.py (hunk 1 above)
Comment (EthanVanderHorn) [reply nanome/_internal/_structure/_serialization/_workspace_serializer.py]: nooope

File: nanome/api/plugin_instance.py (hunk 2 above)
Comment (EthanVanderHorn) [reply nanome/api/plugin_instance.py]: hmmm

File: nanome/_internal/_structure/_workspace.py [hunk 4]
```diff
@@ -8,11 +8,19 @@ def _create(cls):
         return cls()
 
     def __init__(self):
-        self._position = Vector3(0,0,0)
-        self._rotation = Quaternion(0,0,0,0)
-        self._scale = Vector3(1,1,1)
+        self._position = Vector3()
+        self._rotation = Quaternion()
+        self._scale = Vector3(0.02,0.02,0.02)
```
Comment (Ramjii) [inline nanome/_internal/_structure/_workspace.py]: That's the actual starting scale?

File: nanome/api/plugin_instance.py [hunk 5]
```diff
@@ -205,6 +205,32 @@ def update_node(self, node):
         """
         self._network._send(_Messages.node_update, node)
 
+    def set_menu_transform(self, index, position, rotation, scale):
```
Comment (ajm13) [inline nanome/api/plugin_instance.py]: Should this have defaults for rotation and scale? Do I need to send everything if I just want to position it?

File: nanome/_internal/_structure/_workspace.py (hunk 4 above)
Comment (EthanVanderHorn) [reply nanome/_internal/_structure/_workspace.py]: yeah

File: nanome/api/plugin_instance.py (hunk 5 above)
Comment (EthanVanderHorn) [reply nanome/api/plugin_instance.py]: yes

**Pull request merged**